gc_little_helper_II.user.js

```diff
@@ -1394,6 +1394,7 @@ var mainGC = function() {
         function buildUpHeader(waitCount) {
             if ($('#gc-header, #GCHeader')[0]) {
                 tlc('Header found');
+//xxxx
                 // Integrate old header. closest examples: Dashboard, Owner Dashboard, New Map, My Lists.
                 ($('#gc-header') || $('#GCHeader')).closest('#gc-header-root, #header-root, #root, #app-root').prepend(header_old);
```

Comment:
Ich habe diese Zeile mal durch `$('#gc-header').after(header_old);` ersetzt. Das funktionert bei mir, ohne, dass ich etwas anderes ändern muss. Siehe https://github.com/capoaira/GClh/blob/new-map/header/gc_little_helper_II.user.js

Reply:
Ok, ich meine das hätte ich auch gemacht und auch anderes. Immer wenn ich innerhalb von div.app-content war, hat es geknallt. Zuerst war der Header sichtbar, dann unserer, und dann wurde die Seite komplett weiß und es kamen Fehler in der Konsole. Deine Anpassung funktioniert aber. Ich sollte vielleicht für heute Schluss machen. 🙄

Ich werde die Änderung so machen, auch in meinen anderen branchs, und werde das noch etwas beobachten die nächsten Tage.

Danke für deine Unterstützung.

Hab auch kurz bei dir vorbeigeschaut und musste feststellen, das ich dir noch gar kein Sternchen verpasst hatte. Tse, irgendwie verpennt.

Reply:
> Hab auch kurz bei dir vorbeigeschaut und musste feststellen, das ich dir noch gar kein Sternchen verpasst hatte. Tse, irgendwie verpennt.

Ne, hast du nicht verpennt. Ich habe aber mal den Fehler gemacht etwas in meinen Collector zu committen und hatte den Commit dann in jedem neuen Branch. Da ich das nicht Rückgänig machen konnte, habe ich meinen Fork gelöscht und einen neuen erstellt.